lib/generators/rolemodel/css/base/base_generator.rb

```diff
@@ -6,6 +6,10 @@ class BaseGenerator < Rails::Generators::Base
       include Rolemodel::BundlerHelpers
       source_root File.expand_path('templates', __dir__)
 
+      def install_webpack
+        generate 'rolemodel:webpacker' unless Gem.loaded_specs.has_key?('react-rails')
```

Comment:
Checking if the `react-rails` gem is included before generating `webpacker`. Could you check for the `webpacker` gem instead?

Reply:
I believe I started out by checking for `webpacker`, but it did not work, I think because it is installed even before the webpacker generator is run (not sure why). I found that checking for` react-rails` to be present did the right thing, but agree with you that this could be improved.